### ISSUE TITLE
SUM "Energy Drain" don't count

### ISSUE THREAD

**Issue by ilyabelka** [Elysia-ff/FFXIV_RotationHelper#5]
Faced with the problem that "Energy Drain" is not counted correctly when it is used in the game. The rest of the abilities work fine.

**Comment by Elysia-ff** [comment]
Could you try again with this and let me know if it's working?
https://github.com/Elysia-ff/FFXIV_RotationHelper/releases/tag/v2.2.0.0

**Comment by ilyabelka** [comment]
Ok. I tested some other skills and this is list of non-working spells on SUM:
"Energy Drain" [from 2.1.0.0 - 2.2.1.0]
"Ruin IV" [2.2.1.0]
"Aetherpact" [2.2.1.0]   /   "Devotion" pet skill [2.2.1.0]
"Miasma III" [2.2.1.0]
"Bio III" [2.2.1.0]
"Summon Bahamut" [2.2.1.0]
"Firebird Trance" [2.2.1.0]

Other skills are working properly.
Can it be a problem on my part?

**Comment by Elysia-ff** [comment]
Yes and No.
There are 2 ways to read the logs. v2.2.0.0 uses both way, v2.2.1.0 uses the way A.
But, the working way is different by users.

I couldn't find why, I'm just guessing it's because the order of plugin list..?

**Comment by omashano** [comment]
I am having a similar issue with NIN.
I'm on version 2.3.0.0 and game patch 5.11

I tested out all job actions and the ones not working are

Ninjutsu
Hakke Mujinsatsu
Bhavacakra
Hellfrog Medium
Shadow Fang
Ten Chi Jin
Meisui
Bunshin

While mudras are only recognized if used on their own and not followed by another mudra, for example:
it works if i use Ten followed by Spinning edge (results in failing the ninjutsu), but it doesn't work if i try doing Ten >Chi.

It should be noted that both NIN and SUM have been updated significantly with patch 5.1, and as far as I can tell the abilities not working are the ones that changed the most, so maybe that might have something to do with it.
